src/main/java/com/zepben/cimbend/get/NetworkConsumerClient.kt

```diff
@@ -96,60 +103,117 @@ class NetworkConsumerClient(
         }
     }
 
+    /***
+     * Retrieve the feeder network for the specified [mRID] and store the results in the [service].
+     *
+     * This is a convenience method that will fetch the feeder object, all of the equipment referenced by the feeder (normal state),
+     * the terminals of all elements, the connectivity between terminals, the locations of all elements, the ends of all transformers
+     * and the wire info for all conductors.
+     *
+     * @return The [Feeder], or null if it was nto found.
+     */
+    fun getFeeder(service: NetworkService, mRID: String): GrpcResult<Feeder> {
+        return tryRpc {
+            val feederResponse = getIdentifiedObject(service, mRID)
+            val feeder = feederResponse.result
+
+            if (!feederResponse.wasSuccessful)
+                throw feederResponse.thrown!!
+            else if (feeder == null)
+                return@tryRpc GrpcResult.of(null)
+            else if (feeder !is Feeder)
+                throw ClassCastException("Unable to extract feeder network from ${feeder.typeNameAndMRID()}.")
+
+            validateRpcCall(getIdentifiedObjects(service, service.getUnresolvedReferenceMrids(Resolvers.equipment(feeder))))
+
+            val mRIDs = service.getUnresolvedReferenceMrids(Resolvers.normalEnergizingSubstation(feeder)).toMutableSet()
+
+            feeder.equipment.forEach {
+                if (it is ConductingEquipment) {
+                    it.terminals.forEach { terminal ->
```

Comment:
Looking at this again I think the server always sends terminals when a piece of conducting equipment is requested. That is the "ownedObjects" in the gRPC message. But maybe we don't want that and to just keep it really simple and only send back exactly what was requested?

Reply:
Yes it does always send it back and this is code left over working around another bug that failed to extract the owned objects, I will clean it up

Reply:
Bah, ignore my previous comment. This is getting the connectivity nodes for the terminals, not the terminals themselves

Reply:
isn't this fine? i was under the impression this was retrieving unresolved ConnectivityNodes?

Reply:
On the second point - did we introduce the `ownedObjects` field to solve the ordering problem of terminals and what not - which would now be unnecessary as they store the sequenceNumber.

Reply:
I think that was the primary driver for having them there. The secondary was that they really are part of the same object (same as ends and transformers). If you want to change it make a new task for it.

Reply:
Derp we are all having our moments aren't we.

@kgreav That's the kind of thing I would promote starting a discussion with in the Evolve repo. Guess I should announce what I hoping to use it for, but ultimately I'm hoping that once we agree on something we mark the discussion as solved and create appropriate issues / tasks to get it done. That way there is also a history of *why* we made the change.

Reply:
Yep I figured it'd become a discussion, but I was going to leave it up to you to start a discussion if you deemed it necessary :)